dev/bots/test/analyze_snippet_code_test.dart

```diff
@@ -67,16 +68,19 @@ void main() {
     );
     expect(process.stdout, isEmpty);
     final List<String> stderrLines = process.stderr.toString().split('\n');
-    expect(stderrLines.length, stderrLines.toSet().length, reason: 'found duplicates in $stderrLines');
-    final List<String> stderrNoDescriptions = stderrLines.map(removeLintDescriptions).toList();
+    expect(stderrLines.length, stderrLines.toSet().length,
+        reason: 'found duplicates in $stderrLines');
+    final List<String> stderrNoDescriptions =
+        stderrLines.map(removeLintDescriptions).toList();
     expect(stderrNoDescriptions, <String>[
       ...expectedMainErrors,
       'Found 18 snippet code errors.',
       'See the documentation at the top of dev/bots/analyze_snippet_code.dart for details.',
       '', // because we end with a newline, split gives us an extra blank line
     ]);
     expect(process.exitCode, 1);
-  });
+  }, skip: true); // TODO(scheglov): Restore after landing Dart SDK changes.
+  // https://github.com/flutter/flutter/issues/154413
```

Comment:
To make the bot happy this may have to be all in a single line (there is no line limit in this repo)